models/enums.py

```diff
@@ -0,0 +1,32 @@
+from enum import Enum
+from pathlib import Path
+
+
+class Models(Enum):
+    """
+    Selection of Model
+    """
+
+    KINOVA_DEPRECATED = str(Path("KINOVA_arm_deprecated.bioMod").absolute())
+    KINOVA_LEFT = str(Path("KINOVA_arm_reverse_left.bioMod").absolute())
+    KINOVA_RIGHT = str(Path("KINOVA_arm_reverse_right.bioMod").absolute())
+    KINOVA_MERGE = str(Path("KINOVA_merge.bioMod").absolute())
+    KINOVA_MERGE_INVERSE_KINEMATICS = str(Path("KINOVA_merge_inverse_kinematics.bioMod").absolute())
+    KINOVA_MERGE_WITHOUT_FLOATING_BASE_TEMPLATE = str(Path("KINOVA_merge_without_floating_base_template.bioMod").absolute())
+    KINOVA_MERGE_WITHOUT_FLOATING_BASE_WITH_VARIABLES = str(Path("KINOVA_merge_without_floating_base_template_with_variables.bioMod").absolute())
+    KINOVA_MERGE_WITHOUT_FLOATING_BASE_WITH_6_DOF_SUPPORT_TEMPLATE = str(Path("KINOVA_merge_without_floating_base_with_6_dof_support_template.bioMod").absolute())
+    KINOVA_MERGE_WITHOUT_FLOATING_BASE_WITH_6_DOF_SUPPORT_VARIABLES = str(Path("KINOVA_merge_without_floating_base_with_6_dof_support_template_with_variables.bioMod").absolute())
+    KINOVA_MERGE_WITHOUT_FLOATING_BASE_WITH_ROTOTRANS_SUPPORT_TEMPLATE = str(Path("KINOVA_merge_without_floating_base_with_rototrans_template.bioMod").absolute())
+    KINOVA_MERGE_WITHOUT_FLOATING_BASE_WITH_ROTOTRANS_SUPPORT_VARIABLES = str(Path("KINOVA_merge_without_floating_base_with_rototrans_template_with_variables.bioMod").absolute())
```

Comment:
Instead of KINOVA_MERGE, write WU_AND_KINOVA and place this category after Wu models in another section.

Reply:
Group them by two, template and variables and add space after a group

Reply:
done